geospaas_harvesting/ingesters.py

```diff
@@ -543,7 +543,16 @@ def _get_parameter_names(self, dataset):
 
     def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         raw_attributes = self._get_raw_attributes(dataset_info)
-        return self._metadata_handler.get_parameters(raw_attributes)
+        normalized_attributes = self._metadata_handler.get_parameters(raw_attributes)
+
+        if dataset_info.startswith('http'):
+            normalized_attributes['geospaas_service'] = HTTP_SERVICE
+            normalized_attributes['geospaas_service_name'] = HTTP_SERVICE_NAME
+        else:
+            normalized_attributes['geospaas_service'] = LOCAL_FILE_SERVICE
+            normalized_attributes['geospaas_service_name'] = FILE_SERVICE_NAME
```

Comment:
How about the cases that someone incorrectly puts a FTP address in the entry of harvesting with this ingester? Then that FTP address is stored in the database as a local address! It should not work like that!

Reply:
There will be an error because the netcdf4 library can't open ftp URLs